Dockerfile-kyrix-ubuntu

```diff
@@ -0,0 +1,52 @@
+FROM ubuntu:16.04
```

Comment:
just a q: if alpine is always better, do we need to keep this ubuntu dockerfile?

Reply:
DONE: explained in the Dockerfiles :-)